nixos/modules/virtualisation/podman.nix

```diff
@@ -121,6 +121,9 @@ in
 
       systemd.packages = [ cfg.package ];
 
+      systemd.services.podman.path = [
+        pkgs.iproute2
```

Comment:
This should probably go in the wrapper rather than here.

Reply:
Non-root podman doesn't manipulate network objects though, so I figured this place is appropriate. It's efficient from a technical standpoint. Let me know if you still want to move it.

Reply:
I don't disagree but I'd prefer to have the runtime deps handled consistently. Having a single dep here and only being added to the service path makes it easy to overlook and more complicated than it needs to be.

Reply:
It's also worth noting that we want `iproute2` in `$PATH` for non-nixos root podman.

Reply:
Done.